otherlibs/Makefile.otherlibs.common

```diff
@@ -61,6 +62,8 @@ all: $(STUBSLIB) $(LIBNAME).cma $(CMIFILES)
 allopt: $(STUBSLIB) $(LIBNAME).cmxa $(LIBNAME).$(CMXS) $(CMIFILES)
 opt.opt: allopt
 
+noop:
```

Comment:
Would there be an easy way to define a default `noop` rule in `Makefile.common` that other makefiles can override? Or maybe you prefer to do it manually because you want to ensure that "children noops" are defined in the right way?